solidity/contracts/libraries/operator/Groups.sol

```diff
@@ -85,7 +85,7 @@ library Groups {
         bytes memory groupPubKey,
         address[] memory members,
         bytes memory misbehaved
-    ) internal {
+    ) public {
```

Comment:
From #1863:

> From the peanut gallery, this change looks odd without some sort of ACL. Anyway, flagging it for proper review

Reply:
The rationale is that this is one of the larger internal functions in `Groups`, and is only called once per group so the gas cost impact of `DELEGATECALL` instead of inlining should not be excessive. Because `Groups` is a library, it either gets all applicable ACL from the contract calling it or I'm thoroughly misunderstanding something.

Reply:
> Because Groups is a library, it either gets all applicable ACL from the contract calling it or I'm thoroughly misunderstanding something.

Yep, we deal with deposits this way in tBTC as well.